src/source_map_utils.ts

```diff
@@ -0,0 +1,72 @@
+import {SourceMapConsumer, SourceMapGenerator} from 'source-map';
+
+const INLINE_SOURCE_MAP_REGEX =
+    new RegExp('^//# sourceMappingURL=data:application/json;base64,(.*)$', 'm');
+
+export function containsInlineSourceMap(source: string): boolean {
+  return getInlineSourceMapCount(source) > 0;
+}
+
+export function getInlineSourceMapCount(source: string): number {
```

Comment:
this is only used in tests, is it really important to expose this and test it separately?